src/components/shared/veileder/Veileder.tsx

```diff
@@ -29,23 +27,40 @@ function Veileder({
 
     return (
         <div
-            className={cn(styles.veilederWrapper, { [styles.disableBorder]: !border }, { [styles.flexWrap]: flexWrap })}
+            className={cn(
+                'mb-4 flex items-center justify-center print:hidden',
+                {
+                    'disableBorder [&_p:not(:last-of-type)]:mb-3 [&_p]:m-0': !border,
+                },
+                {
+                    'flexWrap mb-12': flexWrap,
+                },
+            )}
         >
-            <GuidePanel className={styles.veilederPanel} illustration={illustration}>
+            <GuidePanel
+                className={cn(
+                    'ml-12 mr-12 flex w-full items-center [&_p]:text-base',
+                    'last-of-type:[.disableBorder_&>div]:border-none',
+                    'first-of-type:[.flexWrap_&>div]:relative first-of-type:[.flexWrap_&>div]:top-0 first-of-type:[.flexWrap_&>div]:-mr-6',
+                    'last-of-type:[.flexWrap_&>div]:flex last-of-type:[.flexWrap_&>div]:flex-wrap last-of-type:[.flexWrap_&>div]:content-center last-of-type:[.flexWrap_&>div]:p-0',
+                    'max-[425px]:first-of-type:[.flexWrap_&>div]:ml-12 max-[768px]:[.flexWrap_&]:m-0 max-[425px]:[.flexWrap_&]:flex-col max-[425px]:[.flexWrap_&]:p-0',
```

Comment:
Det blei litt meget her, så får sjå om eg kan dele den opp i fleire komponentar f.eks.

Reply:
hmm ja dette var litt voldsomt. :P

1. er det noe vi kan overskrive med tokens istedenfor? Dette er den offisielle måten å customize sånt på:

https://aksel.nav.no/komponenter/core/guidepanel#hb02ac6889655

2. Det er lov å ha en css-module dersom tailwinden blir for kompleks. Det er ingen "kost" å ha en CSS-module eller to mtp. bygg-system etc etc. :smile: